src/ui/components/shared/SharingModal/PrivacyDropdown.tsx

```diff
@@ -10,8 +10,7 @@ import { isPublicDisabled } from "ui/utils/org";
 
 const WorkspacePrivacySummary = ({ workspace: { name } }: { workspace: Workspace }) => (
   <span>
-    {`Members of `}
-    <span className="underline">{name}</span>
```

Comment:
I prefer having `Members of` since I think that makes it the grouping consistent (Anyone..., Only people..., Members of...). I'll leave it to you!